paths/intro-to-github/02-join-class-repo.md

```diff
@@ -11,7 +11,7 @@ sidebar:
 main-content:
   includes:
   - activity/intro-to-github/01-join-intro-repo.md
-show-me-how: "images/gifs/intro/join-class-repo.gif"
+show-me-how: "images/gifs/intro/join-repo-make-issue.gif"
```

Comment:
I would rename to `join-repo-make-issue-introduction.gif` for consistency with `join-repo-make-issue-pages.gif `.